src/netspresso_trainer/models/backbones/experimental/shufflenetv2.py

```diff
@@ -40,6 +40,12 @@ def __init__(
             self.stage_out_channels = [-1, 24, 176, 352, 704]
         elif model_size == '2.0x': 
             self.stage_out_channels = [-1 ,24, 244, 488, 976]
+        else:
+            raise NotImplementedError(
```

Comment:
@illian01 This change implements a check for supported model_size values ('0.5x', '1.0x', '1.5x', '2.0x'). It raises a NotImplementedError if an unsupported value is provided, preventing potential issues downstream and improving user experience by providing clear feedback on valid options.

Reply:
Thank you for your detailed update.
I agree with it.